azero/contracts/most/lib.rs

```diff
@@ -646,6 +652,30 @@ pub mod most {
             }
         }
 
+        #[ink(message)]
+        pub fn needs_signature(
+            &self,
+            request_hash: HashedRequest,
+            account: AccountId,
+        ) -> Result<bool, MostError> {
+            if !self.is_in_committee(self.get_current_committee_id()?, account) {
```

Comment:
This my fail for older requests which still need to be signed - remove that check or make needs_signature take correct committe_id as argument.

Reply:
changed